screens/WalletNavigator/screens/SettingsScreen/SettingsScreen.tsx

```diff
@@ -1,36 +1,98 @@
 import * as React from 'react'
 import tailwind from 'tailwind-rn'
-import { createStackNavigator } from '@react-navigation/stack'
-import { Text, View } from '../../../../components/Themed'
+import { useDispatch, useSelector } from 'react-redux'
+import { Text, View, SectionList, TouchableOpacity } from 'react-native'
+import { Ionicons } from '@expo/vector-icons'
 import { translate } from '../../../../translations'
+import { PrimaryColor, PrimaryColorStyle } from '../../../../constants/Colors'
+import { useWalletAPI } from '../../../../hooks/wallet/WalletAPI'
+import { RootState } from '../../../../store'
+import { NetworkName } from '../../../../store/network'
 
 export function SettingsScreen (): JSX.Element {
+  const network = useSelector<RootState, NetworkName | undefined>(state => state.network.name)
+  const WalletAPI = useWalletAPI()
+  const dispatch = useDispatch()
+
   return (
-    <View style={tailwind('flex-1 items-center justify-center')}>
-      <Text style={tailwind('text-xl font-bold')}>
-        {translate('WalletNavigator/SettingsScreen', 'Settings')}
-      </Text>
-      <View style={tailwind('w-4/5 h-px my-8')} lightColor='#eee' darkColor='rgba(255,255,255,0.1)' />
+    <View style={tailwind('flex-1 bg-gray-100')}>
+      <SectionList
+        sections={[
+          {
+            key: 'Network',
+            data: [''],
+            renderItem (): JSX.Element {
+              return RowNetworkItem(network)
+            }
+          },
+          {
+            data: ['EXIT WALLET'],
```

Comment:
Is this label going to be translated?

Reply:
In the end, this wasn't used. I think changed how it was designed halfway through.